doc/env_vars.md

```diff
@@ -112,8 +112,8 @@ You may have noted that some options are not available:
 | `LRSQL_URL_PREFIX` | `urlPrefix` | The prefix of the webserver URL path, e.g. the prefix in `http://0.0.0.0:8080/xapi` is `/xapi`. Used when constructing the `more` value for multi-statement queries. | `/xapi` |
 | `LRSQL_ENABLE_ADMIN_UI` | `enableAdminUi` | Whether or not to serve the administrative UI at `/admin` | `true` |
 | `LRSQL_ENABLE_STMT_HTML` | `enableStmtHtml` | Whether or not HTML data is returned in the LRS HTTP response. If `false` disables HTML rendering even if `LRSQL_ENABLE_ADMIN_UI` is `true`. In that case the UI will not display the Statement Browser feature. | `true` |
-| `LRSQL_OIDC_ISSUER` | `oidcIssuer` | OIDC Issuer address used for [discovery](https://openid.net/specs/openid-connect-discovery-1_0.html#ProviderConfig) | Not set |
-| `LRSQL_OIDC_CONFIG` | `oidcConfig` | Custom OIDC configuration location | Not set |
-| `LRSQL_JWKS_URI` | `jwksUri` | Custom JWKS [keyset](https://datatracker.ietf.org/doc/html/rfc7517#section-5) location | Not set |
+| `LRSQL_OIDC_ISSUER` | `oidcIssuer` | OIDC Issuer address used for [discovery](https://openid.net/specs/openid-connect-discovery-1_0.html#ProviderConfig). Will enable OIDC if present. | Not set |
+| `LRSQL_OIDC_AUDIENCE` | `oidcAudience` | Optional OIDC audience for token claim verification. | Not set |
+| `LRSQL_OIDC_VERIFY_REMOTE_ISSUER` | `oidcVerifyRemoteIssuer` | Verify on startup that the issuer in remote configuration matches `LRSQL_OIDC_ISSUER` (if provided). | `true` |
```

Comment:
Wait I thought you were just going to keep `LRSQL_OIDC_ISSUER`, i.e. only have one OIDC-related env var.

Reply:
The vars I removed were alternates for `LRSQL_OIDC_ISSUER` (`LRSQL_OIDC_CONFIG` and `LRSQL_JWKS_URI`). If you just provide issuer, you can use the LRS as an OIDC resource server (for xAPI/admin api). If you provide `LRSQL_OIDC_AUDIENCE`, it will additionally verify token audience (this will be recommended in the docs, which are pending). `LRSQL_OIDC_VERIFY_REMOTE_ISSUER` toggles an additional sanity check that the issuer you put in matches the one in the remote OIDC config.

Reply:
Maybe also add in the `LRSQL_OIDC_AUDIENCE` and `LRSQL_OIDC_VERIFY_REMOTE_ISSUER` descriptions that they will be unused if `LRSQL_OIDC_ISSUER` is not set.

Reply:
thanks, I meant to do that yesterday

Reply:
https://github.com/yetanalytics/lrsql/pull/207/commits/e8c90be1e5b0218a5aa3bf6df72cc44230c3a622